dspace-server-webapp/src/main/java/org/dspace/app/rest/DiscoveryRestController.java

```diff
@@ -201,6 +201,8 @@ public RepresentationModel getFacetValues(@PathVariable("name") String facetName
         FacetResultsRest facetResultsRest = discoveryRestRepository
             .getFacetObjects(facetName, prefix, query, dsoTypes, dsoScope, configuration, searchFilters, page);
 
+        facetResultsRest.setConfiguration(configuration);
```

Comment:
Could you please move this setter to the method that is responsible for creating the `FacetResultsRest` object? That method is the following `org.dspace.app.rest.converter.DiscoverFacetResultsConverter#convert`